src/policy/concrete.rs

```diff
@@ -242,6 +245,7 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
                     });
                 (all_safe, atleast_one_safe && all_non_mall)
             }
+            Policy::TxTemplate(_) => (true, true),
```

Comment:
again not so sure about the safe flag here.

Reply:
lifting this to the main thread